src/lib/cli-util.js

```diff
@@ -0,0 +1,41 @@
+var size = require('window-size')
+var availableWidth = size.width || /*istanbul ignore next */ 80
+var ui = require('cliui')({width: availableWidth})
+
+function push(output, columns) {
+  var _output = [].concat(output)
+
+  var padding = {top: 0, right: 2, bottom: 0, left: 0}
+  var width = _output.reduce(
+    function getMaxWidth(previous, current) {
+      return Math.max(padding.left + current.length + padding.right, previous)
+    }, 0)
+
+  var _columns = columns || Math.floor(availableWidth / width)
+  var cellMapper = getOutputCellMapper(Math.floor(availableWidth / _columns), padding)
+
+  while (_output.length) {
+    ui.div.apply(ui, _output.splice(0, _columns).map(cellMapper))
```

Comment:
Yeah, that gave me a bit of a headache, I was hoping that I could somehow flush the output earlier, but after a quick glance over `cliui`'s sources, I guess this is not possible without creating a new instance of it every time the function now called `push` is invoked.

Maybe this is something to keep in mind for later, if we'd have real hard calculations going on, I'd prefer to print stuff asap, but in this case I guess it's of minor importance.